src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -25,6 +25,7 @@
     <ProjectReference Include="..\Sarif.Converters\Sarif.Converters.csproj" />
     <ProjectReference Include="..\Sarif.Driver\Sarif.Driver.csproj" />
     <ProjectReference Include="..\Sarif\Sarif.csproj" />
+    <ProjectReference Include="..\Sarif.UnitTests\Sarif.UnitTests.csproj" />
```

Comment:
>UnitTests [](start = 56, length = 9)

If `Sarif.UnitTests` has code that `Sarif.FunctionalTests` needs, then that code belongs in `Sarif.TestUtilities`, and you should reference that project instead.